app.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+// Global Variables
+var images = [
+  new Image('img/bag.jpg', 'bag'),
```

Comment:
I love the way you are calling your constructor inside the array declaration.  Great pattern.